twarc/command2.py

```diff
@@ -518,12 +524,16 @@ def timelines(
 
         users = []
         try:
-            data = ensure_flattened(json.loads(line))
-            users = set([t["author"]["id"] for t in ensure_flattened(data)])
+            # Handle both quoted user_ids and json tweet data
+            json_data = json.loads(line)
+            try:
+                users = set([t["author"]["id"] for t in ensure_flattened(json_data)])
+            except ValueError:
+                users = set(json_data)
```

Comment:
I see this was introduced to guard against usernames that are quoted? Isn't there a danger here that any JSON that is passed in will be sent along? Maybe we could check that `json_data` is a string and log an error ad skip if it's not?

Reply:
That's a good point, I've fixed that now.